src/storage/data_table.cpp

```diff
@@ -203,11 +205,14 @@ bool DataTable::SelectIntoBuffer(transaction::TransactionContext *const txn, con
       }
       StorageUtil::CopyAttrIntoProjection(accessor_, slot, out_buffer, i);
     }
+
+    // TODO(Matt): might not need to read visible in the loop (move after?) but not confident without large random tests
```

Comment:
This wasn't a change made by me, it came in when I pulled in cmu-db master. I think it's probably best for Matt to update this?

Reply:
You can remove this comment.  Matt and I talked it looking at the index branch.